docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/customizing-policy.adoc

```diff
@@ -0,0 +1,28 @@
+= Customizing Policy
+
+An Enterprise Contract Policy defines which checks are performed by EC during integration tests and
+release. If EC reports violations, the best course of action, in most cases, is to adjust the build
+process to remedy the violation. For example, if EC reports that a certain required scanner was not
+executed, add the scanner to your build pipeline.
+
+However, there are times where remedying the violation is not an option. In this case, customizing
+the Policy is the only option to waive the failing check.
+
+First, xref:/advanced-how-tos/managing-compliance-with-ec/identifying-policy-in-use.adoc[identify]
+which Policy is being used to verify the Components in your Application.
+
+Second, determine if this is a Policy you can modify. Policies are often shared between different
+users. Adding an exception to a shared Policy may not be in the best interest of all users. Creating
+a new Policy for your particular use case may be a better approach. You can copy an existing Policy
+as a starting point.
+
+Third, update your Policy to exclude a certain violation. See the
+link:https://enterprisecontract.dev/docs/ec-cli/main/configuration.html[Policy Configuration] docs
+for details and examples.
+
+Fourth, update your xref:/how-tos/testing/integration/editing.adoc[integration tests] and the
+xref:/advanced-how-tos/releasing/create-release-plan-admission.adoc[ReleasePlanAdmission] to use the
+new Policy.
+
+Finally, if needed, re-run the xref:/how-tos/testing/integration/rerunning.adoc[integration tests]
+or xref:/advanced-how-tos/releasing/create-release.adoc[release].
```

Comment:
Can we switch to using a procedure like we do elsewhere? I don't think that we use the ordinal numbering words elsewhere in the repo.

Reply:
Sure thing! Turned this into an unordered list instead.